.github/workflows/reusable-agent-build-container-images.yml

```diff
@@ -15,17 +15,24 @@ on:
         description: "AWS region for remote builds."
         type: string
 
-      aws_private_key_name:
-        description: "Name of a private key that is used to connect to AWS EC2 instances that will be created by this workflow."
-        type: string
-
       cicd_workflow:
         description: "Additional string that has to differentiate AWS object from another workflows."
         type: string
+      
+      builder_name:
```

Comment:
Can you please add some description on what this parameter represents and valid / possible values?